modules/ocf_desktop/files/xsession/print-notify-listener

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euo pipefail
+
+function clean_up {
+    jobs -pr | xargs pgrep -P | xargs sudo -u ocfbroker kill
+    exit 0
+}
+
+trap clean_up INT TERM EXIT
+
+pipe="$HOME/tmp/print-notify"
```

Comment:
I think still putting this as a dot file or hiding it a bit more would make more sense, otherwise I could see (a few) users getting confused by finding it and then potentially deleting it.

Reply:
I think `/run/user/$UID/` is the appropriate place for these kinds of fifos and sockets.